hypha/apply/funds/templates/funds/includes/table_filter_and_search.html

```diff
@@ -66,6 +66,11 @@ <h2 class="text-xl mb-2">{{ heading }}</h2>
                     <a href="{% modify_query archived='1' %}">{% trans "Show archived" %}</a>
                 {% endif %}
             {% endif %}
+            {% with request.get_full_path as path %}
+                {% if 'submission' in path and can_export %}
+                    <a href="{{path}}{% if '?' in path %}&{%else%}?{% endif %}export=true" class="button--primary">{% trans 'Export CSV' %}</a>
```

Comment:
It should be "button button--primary".

Reply:
So many redundant classes but OK, I did it : )